low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3.6
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import configparser
+import sys
+import os
+import consul
+
+from cortx.sspl.bin.sspl_constants import (CONSUL_HOST, CONSUL_PORT, PRODUCT_NAME,
+    PRODUCT_FAMILY, file_store_config_path, storage_type, server_type, SSPL_STORE_TYPE)
+
+
+# TODO: Update sensor information in ConfStore
+# from cortx.utils.conf_store import Conf
+
+def update_sensor_info():
+    config = configparser.ConfigParser(allow_no_value=True)
```

Comment:
Why are we not using ConfStore like used in the PostInstall ?

Reply:
Done.